dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.wsSecurity1.1.internal.jaxws-4.0.feature

```diff
@@ -4,6 +4,7 @@ visibility=private
 singleton=true
 -features=io.openliberty.appSecurity-5.0, \
  io.openliberty.servlet.api-6.0; apiJar=false, \
+ com.ibm.websphere.appserver.servlet-6.0, \
```

Comment:
Why are we adding the Servlet 6.0 feature to this GA feature?

Reply:
xmlWS-4.0 now depends on 6.0 and tolerates 6.1 and with public features tolerates not be transitive, we need to declare 6.0 here to say for EE 10 we want servlet-6.0.